app/javascript/src/LoginPage/loginForms/FormGroups.jsx

```diff
@@ -82,11 +87,17 @@ const EmailField = ({inputProps}: {inputProps: InputProps}) => {
 }
 
 const PasswordField = ({inputProps}: {inputProps: InputProps}) => {
+  const isPasswordConfirmation = inputProps.isPasswordConfirmation
```

Comment:
This doesn't belong here IMO, it adds new responsibilities that makes a Password field more complex that it should. I think there should be a `ConfirmPasswordField` instead.